flux-fixpoint/src/lib.rs

```diff
@@ -156,6 +158,9 @@ impl KVar {
 
 impl<Tag: fmt::Display> fmt::Display for Task<Tag> {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        if self.scrape_quals {
+            writeln!(f, "(fixpoint \"--scrape=both\")")?;
```

Comment:
Is `both` necessary to verify the tests we have?

Reply:
Good point -- not really. `--both` verifies these examples if you remove ALL the default qualifiers, but `--head` suffices with the default qualifers (as we need something like `x <= y`)

Reply:
So for now, I'd like to leave `both` as default. Another option is to make `scrape_quals` not a `bool` but something like `No|Head|Both` ?